client/src/app/components/how-it-works/how-it-works.component.scss

```diff
@@ -0,0 +1,23 @@
+$grayColor: #B0BFC7;
```

Comment:
Please define it in the `vars.scss` file (in `assets` directory) and `import` it here if you need it.